### PR TITLE
bug fixes

### LINKED ISSUE #97: [PE-D] App crashes after `edit` is input in reco mode.
When the user inputs `edit` without any indexes, the app continues to accept input, however crashes after the editing is finished.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: CabbageTime/ped#7

### LINKED ISSUE #118: [PE-D] Empty name field causes program to crash
For editing recos, even if the name of the recommendation is not given, the program continues to ask for the other parameters like which category to change, and eventually crashes. This could be fixed by checking for empty string or end of line after the word 'edit'.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: fupernova/ped#8

### LINKED ISSUE #120: [PE-D] Duplicate review entries allowed if transferred
If transferred over from recos, duplicate reviews are allowed. This can be done by additional checks.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: fupernova/ped#10

### PR DESCRIPTION
**Pull request by Krithigha24**
Bug fixes resolved:
Duplicate review entries allowed if transferred Fixes #120 
App crashes after the edit is input in reco mode.
Fixes #97 Fixes #118

### CODE REVIEW

File: src/main/java/seedu/connoisseur/commands/RecommendationList.java [hunk 1]
```diff
@@ -169,6 +174,19 @@ public void addRecommendationDetails() throws DuplicateException, EmptyInputExce
...
+                ui.println(CHANGE_RECO_TITLE);
+                ui.println(ABANDON_RECO);
+                input = ui.readCommand().toLowerCase().trim();
+                if (input.equals("y")) {
+                    return;
+                } else if (input.equals("n")) {
+                    continue;
+                } else {
+                    ui.println(INVALID_COMMAND);
+                    return;
```
Comment (huachen24) [inline src/main/java/seedu/connoisseur/commands/RecommendationList.java]: In this case anything other than 'n' will abandon the recommendation. Shall we have a loop to allow multiple tries for inputting y/n?

File: src/main/java/seedu/connoisseur/messages/Messages.java [hunk 2]
```diff
@@ -90,11 +90,12 @@
     public static final String CURRENT_DIRECTORY = "Present project directory is: " + System.getProperty("user.dir");
 
     // General
-    public static final String DUPLICATE_REVIEW = "There is a review in your list with the same title:";
+    public static final String DUPLICATE_REVIEW = "There is a review in your review list with the same title:";
     public static final String DUPLICATE_RECOMMENDATION = "There is a recommendation in your list with the same title:";
```
Comment (huachen24) [inline src/main/java/seedu/connoisseur/messages/Messages.java]: duplicate review says review list but duplicate recommendation only says list. Shall we standardize it?

File: src/main/java/seedu/connoisseur/commands/RecommendationList.java (hunk 1 above)
Comment (EmilyTJX) [reply src/main/java/seedu/connoisseur/commands/RecommendationList.java]: sounds good

File: src/main/java/seedu/connoisseur/messages/Messages.java (hunk 2 above)
Comment (EmilyTJX) [reply src/main/java/seedu/connoisseur/messages/Messages.java]: do we use reviewS list?

File: src/main/java/seedu/connoisseur/commands/RecommendationList.java (hunk 1 above)
Comment (Krithigha24) [reply src/main/java/seedu/connoisseur/commands/RecommendationList.java]: image
ok i have resolved this alr!

File: src/main/java/seedu/connoisseur/messages/Messages.java (hunk 2 above)
Comment (Krithigha24) [reply src/main/java/seedu/connoisseur/messages/Messages.java]: image
standardised!

**Review by huachen24 — APPROVED** [review]

**Pull request merged**